src/components/game/controller.js

```diff
@@ -9,17 +10,33 @@ export const calculateWinner = (squares) => {
     [0, 4, 8],
     [2, 4, 6],
   ];
-  for (let i = 0; i < lines.length; i++) {
-    const [a, b, c] = lines[i];
-    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
-      return { winnerCombination: [a, b, c], winnerName: squares[a] };
+  /* For loop that goes through & checks that the array has the
+   winner combination positions in it */
+  for (
+    let winnerArray = 0;
```

Comment:
The variable name here says that it's a winnerArray but defines an integer value

Reply:
Changed variable to something more appropriate